packages/kit/test/apps/basics/test/test.js

```diff
@@ -1622,10 +1621,7 @@ test.describe('Routing', () => {
 
 		try {
 			await page.goto(`/routing?port=${port}`);
-			await Promise.all([
-				page.click(`[href="http://localhost:${port}"]`),
-				page.waitForURL(`http://localhost:${port}/`)
-			]);
+			await page.locator(`[href="http://localhost:${port}"]`).click();
```

Comment:
What is this even testing (unrelated to the changes)? I don't see an assertion here.

Reply:
Good question. Looks like that test was added in https://github.com/sveltejs/kit/pull/1487. I haven't gotten a chance to take a look at it yet and figure out if it might actually be testing something

Reply:
I've changed this back to the prior code and added a comment describing the test